### PR TITLE
MOSIP-15287 openapi.json file generated

### PR DESCRIPTION
**Pull request by kameshsr**
openapi.json file generated during build time after running  `mvn clean verify -Dgpg.skip -DskipTests -P openapi-doc-generate-profile` this.

### CODE REVIEW

File: resident/resident-service/pom.xml [hunk 1]
```diff
@@ -147,7 +147,7 @@
 		<dependency>
 			<groupId>io.mosip.commons</groupId>
 			<artifactId>commons-packet-manager</artifactId>
-			<version>${commons.packet.manager.version}</version>
+			<version>1.1.5.3</version>
```
Comment (vishwa-vyom) [inline resident/resident-service/pom.xml]: why is this version changed ?

File: resident/resident-service/src/main/java/io/mosip/resident/ResidentBootApplication.java [hunk 2]
```diff
@@ -6,6 +6,7 @@
 @SpringBootApplication(scanBasePackages = { "io.mosip.resident.*", "io.mosip.kernel.core.*",
 		"io.mosip.kernel.crypto.jce.*", "io.mosip.commons.packet.*", "io.mosip.kernel.keygenerator.bouncycastle.*",
 		"${mosip.auth.adapter.impl.basepackage}" })
+
```
Comment (vishwa-vyom) [inline resident/resident-service/src/main/java/io/mosip/resident/ResidentBootApplication.java]: why is this extra line required ?

File: resident/resident-service/src/test/resources/application.properties [hunk 3]
```diff
@@ -33,65 +170,120 @@ resident.clientId=mosip-resident-client
...
-IDAUTHCREATEVID=http://idrepo-vid-service/idrepository/v1/vid
-IDAUTHREVOKEVID=http://idrepo-vid-service/idrepository/v1/vid/deactivate
-#Notification service
-IDREPOGETIDBYUIN=http://idrepo-identity-service/idrepository/v1/identity/idvid
-IDREPOGETIDBYRID=http://idrepo-identity-service/idrepository/v1/identity/idvid
-GETUINBYVID=http://idrepo-vid-service/idrepository/v1/idvid
-resident.notification.emails=mosiptestuser@gmail.com
-resident.notification.message=Notification has been sent to the provided contact detail(s)
+
+#------Temporary | remove this --------------------------
```
Comment (vishwa-vyom) [inline resident/resident-service/src/test/resources/application.properties]: Why does few comment says remove this ? do we need all these properties to run the open api plugin ?

File: resident/resident-service/pom.xml [hunk 4]
```diff
@@ -147,7 +147,7 @@
 		<dependency>
```
Comment (vishwa-vyom) [inline resident/resident-service/pom.xml]: Always have the jira Id with the exact format in commit messages and it the PR titile

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: resident/resident-service/src/test/resources/application.properties (hunk 3 above)
Comment (kameshsr) [reply resident/resident-service/src/test/resources/application.properties]: No, we don't need these i have removed this in latest commit.

File: resident/resident-service/pom.xml (hunk 1 above)
Comment (kameshsr) [reply resident/resident-service/pom.xml]: Fixed this with default version.

File: resident/resident-service/src/main/java/io/mosip/resident/ResidentBootApplication.java (hunk 2 above)
Comment (kameshsr) [reply resident/resident-service/src/main/java/io/mosip/resident/ResidentBootApplication.java]: Fixed this in latest commit.

File: resident/resident-service/pom.xml (hunk 4 above)
Comment (kameshsr) [reply resident/resident-service/pom.xml]: yes added.

Comment (vishwa-vyom) [reply resident/resident-service/pom.xml]: ok resolved

File: resident/resident-service/src/test/resources/application.properties (hunk 3 above)
Comment (vishwa-vyom) [reply resident/resident-service/src/test/resources/application.properties]: ok resolved

File: resident/resident-service/src/main/java/io/mosip/resident/ResidentBootApplication.java (hunk 2 above)
Comment (vishwa-vyom) [reply resident/resident-service/src/main/java/io/mosip/resident/ResidentBootApplication.java]: I still see this in the latest commit

File: resident/resident-service/pom.xml (hunk 1 above)
Comment (vishwa-vyom) [reply resident/resident-service/pom.xml]: ok resolved

File: resident/resident-service/src/main/java/io/mosip/resident/ResidentBootApplication.java (hunk 2 above)
Comment (kameshsr) [reply resident/resident-service/src/main/java/io/mosip/resident/ResidentBootApplication.java]: Yes sir I fixed this in latest commit.

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**